src/test/java/com/exasol/errorcodecrawlermavenplugin/crawler/ErrorMessageDeclarationCrawlerTest.java

```diff
@@ -45,6 +54,31 @@ void testCrawlValidCode() {
         );
     }
 
+    @Test
+    void testSubProjectCrawlValidCode() throws IOException {
+        Path subProjectDir = projectDir.resolve("sub-project");
+        Path subProjectTestSrcJava = subProjectDir.resolve(Path.of("src", "test", "java"));
+        Path subProjectTestSrcPackage = subProjectTestSrcJava
+                .resolve(Path.of("com", "exasol", "errorcodecrawlermavenplugin", "examples"));
+        subProjectTestSrcPackage.toFile().mkdirs();
+        ErrorMessageDeclarationCrawler subProjectDeclarationCrawler = new ErrorMessageDeclarationCrawler(
+                subProjectDir.getParent(), subProjectDir, emptyList(), 11, Collections.emptyList());
+        Files.copy(Path.of(TEST_DIR).resolve("Test1.java"), subProjectTestSrcPackage.resolve("Test1.java"), StandardCopyOption.REPLACE_EXISTING);
+        final Path path = projectDir.relativize(subProjectTestSrcPackage.resolve("Test1.java"));
```

Comment:
Rename to `expectedPath`. Also better define the path without using `relativize` to make it clear which path is expected.